circle.yml

```diff
@@ -1198,37 +1198,34 @@ jobs:
 
   system-tests-node-modules-install:
     <<: *defaults
+    resource_class: medium+ ## trying this out to see if it makes a difference since this is a critical blocking job
```

Comment:
Did this yield any positive results? Looking at the output it seems like it's not making full use of `medium+`: https://app.circleci.com/pipelines/github/cypress-io/cypress/32461/workflows/d2c7ea79-3f7e-4c0f-a50a-fa65b8853ce7/jobs/1276173/resources